earth_enterprise/src/third_party/apache2/apache-ge-httpd.conf.in-handle_http_errors.patch

```diff
@@ -1,18 +1,18 @@
 diff -uNr httpd-2.2.29.orig/docs/conf/httpd.conf.in httpd-2.2.29/docs/conf/httpd.conf.in
 --- httpd-2.2.29.orig/docs/conf/httpd.conf.in	2015-08-27 12:03:00.276038659 -0700
 +++ httpd-2.2.29/docs/conf/httpd.conf.in	2015-08-27 12:12:32.168159817 -0700
-@@ -289,6 +289,7 @@
+@@ -285,6 +285,7 @@
      # directives as to Alias.
      #
      ScriptAlias /cgi-bin/ "@exp_cgidir@/"
 +    Alias /error_handler "/opt/google/gehttpd/cgi-bin/"
```

Comment:
The added line above in the patch needed to be removed as it causing custom error handling to be not working: at least when launching cutter while cutter service is disabled. Instead of showing 403 error (as cutter service is disabled by default). it was showing different/unexpected content.